helm-chart/templates/api-secrets.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: api
+  labels:
+    {{- include "avatar.labels" . | nindent 4 }}
+type: Opaque
+
+data:
+  pepper: {{ .Values.api.pepper | b64enc | quote }}
+  authjwt_secret_key: {{ .Values.api.authjwtSecretKey | b64enc | quote }}
+  file_encryption_key: {{ .Values.api.fileEncryptionKey | b64enc | quote }}
```

Comment:
`b64enc` looks handy! I guess you doubled checked that you indeed need to b64 encode the encryption key, which is already in base64 ?

Reply:
yes :)